x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -80,6 +81,15 @@ export class HeadlessChromiumDriver {
     logger.info(`handled ${interceptedCount} page requests`);
   }
 
+  public async getMetrics(): Promise<PerformanceMetrics> {
+    const puppeteerMetrics = await this.page.metrics();
```

Comment:
Wow, can't believe it's that simple of a call

Reply:
Yeah... that kind of tells me the peripheral changes here should be a separate PR. I'm actually a little hesitant with just adding this call without a followup plan on what to do with the data.

Reply:
https://github.com/elastic/kibana/pull/47081